src/main/groovy/com/morpheusdata/commvault/backup/CommvaultBackupTypeProvider.groovy

```diff
@@ -33,7 +38,8 @@ class CommvaultBackupTypeProvider extends AbstractBackupTypeProvider {
 	 */
 	@Override
 	String getCode() {
-		return "commvaultBackupTypeProvider"
```

Comment:
After converting to an abstract class `getCode()` and `getName()` can be removed.